signac_dashboard/dashboard.py

```diff
@@ -501,6 +505,10 @@ def update_cache(self):
                            map(lambda x: x[1], members)):
             func.cache_clear()
 
+    def __call__(self, environ, start_response):
```

Comment:
Is that the standard API for this?

Reply:
Yes, WSGI applications are always callables with this signature. https://en.wikipedia.org/wiki/Web_Server_Gateway_Interface#Examples

Reply:
Perfect!